app/controllers/participant_bulk_actions_controller.rb

```diff
@@ -0,0 +1,93 @@
+class ParticipantBulkActionsController < ParticipantsController
+  before_filter :fetch_participants
+  before_filter :check_authorization
+  
+	def send_email
+		@emails = @participants.collect(&:email).flatten.compact.uniq
+		if @emails.empty?
+			flash[:error] = "You must select at least one record with an e-mail address."
+			render :text => flash[:error], :status => 200
+		else
+      # flash[:notice] = "Sent #{@template.pluralize(@emails.count, "e-mail address")} to your e-mail program."
+			render :js => "window.location.href = 'mailto:#{@emails.join(",")}';"
```

Comment:
I'm not sure about rendering in views, but I'm guessing the sanitizer is going over this.